.buildkite/commands/build-for-testing.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eu
+
+# FIXIT-13.1: Temporary fix until we're on the Xcode 13.1 VM
+echo "--- :rubygems: Fixing Ruby Setup"
+gem install bundler
+
+echo "--- :rubygems: Setting up Gems"
+restore_cache "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
+install_gems
+
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
+
+echo "--- :writing_hand: Copy Files"
+cp -v fastlane/env/project.env.example .configure-files/project.env
```

Comment:
I don't think this is necessary 🤔 We should always be reading this from the `~/.configure/...` location

https://github.com/woocommerce/woocommerce-ios/blob/81ed7b18078c58d3337f53a91d0f09c802bf3c3f/fastlane/Fastfile#L7-L8


```suggestion
```

Reply:
Buidkite was green on #6098 👍

Reply:
Merged #6098